lib/matplotlib/image.py

```diff
@@ -885,6 +911,7 @@ def __init__(self, ax,
                  cmap=None,
                  norm=None,
                  interpolation=None,
+                 interp_space=None,
```

Comment:
This should go at the end (kwarg only preferably), or we need a deprecation note.